client/src_editor/Editor_Json.re

```diff
@@ -20,6 +20,7 @@ module V1 = {
       json => {
         b_id: json |> field("id", string),
         b_data: json |> field("data", blockDataDecoder),
+        b_deleted: json |> field("deleted", bool),
```

Comment:
could you please make this optional? otherwise it throw with old data.

Reply:
do you mean this?
```reason
json => {
        b_id: json |> field("id", string),
        b_data: json |> field("data", blockDataDecoder),
        b_deleted: json |> optional(field("deleted", bool)),
      };
```
It doesn't like this... it seems like I'd have to change the type to this for this to work... this doesn't seem right?
```reason
  type block = {
    b_id: id,
    b_data: blockData,
    b_deleted: option(bool),
  };
```